packages/hdwallet-native/src/crypto/classes/symmetricCryptoKey.test.ts

```diff
@@ -0,0 +1,45 @@
+import { EncryptionType } from "./encryptionType";
+import { SymmetricCryptoKey } from "./symmetricCryptoKey";
+
+describe("symmetricCryptoKey", () => {
+  const key = new Uint8Array(32).fill(64);
```

Comment:
is this filling the array with a specific bit pattern?  Would a hexadecimal representation be better?

Reply:
this is just for the test, the contents of the key are irrelevant for the test. This is creating a 32-byte array filled with the byte 64 32 times.